src/Bundle/DependencyInjection/Driver/Doctrine/DoctrineODMDriver.php

```diff
@@ -69,6 +69,7 @@ protected function addRepository(ContainerBuilder $container, MetadataInterface
             $unitOfWorkDefinition,
             $this->getClassMetadataDefinition($metadata),
         ]);
+        $definition->addTag('sylius.repository');
```

Comment:
Let's keep repository tag for repositories :) 
```suggestion
        $definition->addTag('sylius.doctrine_driver');
```

Reply:
Hum I've checked and this is not the driver definition, but the repository definition 
https://github.com/loic425/SyliusResourceBundle/blob/317afb718a830576ffe11aef602a5cb0aeed3a5e/src/Bundle/DependencyInjection/Driver/Doctrine/DoctrineODMDriver.php#L66

Reply:
so app.repository.book will be tagged with "sylius.repository" and that makes sense :D